express-notes-api/server.js

```diff
@@ -0,0 +1,83 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+const app = express();
+app.use(express.json());
+
+async function readData() {
+  const file = await readFile('data.json');
+  return await JSON.parse(file);
+}
+const data = await readData();
```

Comment:
I would probably do this at the beginning of each of your endpoints instead of just once here. That way you know you're always reading the most up-to-date data before operating on it.

In this case, it doesn't really matter, though.